cmake.sh

```diff
@@ -25,7 +25,7 @@ SET(Java_JAVAC_EXECUTABLE FALSE CACHE BOOL "" FORCE)
 SET(BUILD_CursesDialog FALSE CACHE BOOL "" FORCE)
 
 # OpenSSL is problematic
```

Comment:
Do you really need to comment this out?

Reply:
This was added to fix the problem commented on 28 Feb.